src/components/Rubric.vue

```diff
@@ -170,7 +213,18 @@
 export default {
   data () {
     return {
-      feedback: {}
+      feedback: {},
+      first_score: null,
+      second_score: null,
+      third_score: null,
+      fourth_score: null,
+      fifth_score: null,
+      first_comment: null,
+      second_comment: null,
+      third_comment: null,
+      fourth_comment: null,
+      fifth_comment: null,
+      overall_comments: null
```

Comment:
Thanks for this! Katie and I were struggling to understand how this relates to the two-way binding.